classes/cookie.php

```diff
@@ -110,15 +128,31 @@ public static function set($name, $value, $expiration = null, $path = null, $dom
 	 * @param   string   $domain     domain of the cookie
 	 * @param   boolean  $secure     if true, the cookie should only be transmitted over a secure HTTPS connection
 	 * @param   boolean  $http_only  if true, the cookie will be made accessible only through the HTTP protocol
+	 * @param   string   $same_site  samesite of the cookie
 	 * @return  boolean
 	 * @uses    static::set
 	 */
-	public static function delete($name, $path = null, $domain = null, $secure = null, $http_only = null)
+	public static function delete($name, $path = null, $domain = null, $secure = null, $http_only = null, $same_site = null)
 	{
 		// Remove the cookie
 		unset($_COOKIE[$name]);
+		
+		if (is_null($same_site)) {
+		    return static::set($name, null, -86400, $path, $domain, $secure, $http_only);
+		}
 
 		// Nullify the cookie and make it expire
-		return static::set($name, null, -86400, $path, $domain, $secure, $http_only);
+		if (PHP_VERSION_ID < 70300) {
+		    return setcookie($name, null, -86400, "{$path}; samesite={$same_site}", $domain, $secure, $http_only);
+		}
+
+		return setcookie($name, null, [
```

Comment:
Fuel coding standards: don't use short array notation for PHP 5.x compatibility.

Reply:
fixed!